src/infra/storage/storage.service.ts

```diff
@@ -1,32 +1,97 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, OnModuleInit } from '@nestjs/common';
+import { Client } from 'minio';
+import { randomUUID } from 'node:crypto';
```

Comment:
Reicht es nicht wenn man import { randomUUID } from 'crypto'; importiert um die build in Libary zu verwenden?

Reply:
done